### PR TITLE
[SPARK-13257] [Improvement] Scala naive Bayes example: check model after loading

### PR DESCRIPTION
**Pull request by movelikeriver**
Refine naive Bayes example by checking model after loading it.
This is Java version, to be consistent with Python version in https://github.com/apache/spark/pull/11125

### CODE REVIEW

File: examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala [hunk 1]
```diff
@@ -15,6 +15,16 @@
  * limitations under the License.
  */
 
+/*
+ * Usage:
```
Comment (holdenk) [inline examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala]: So this usage text probably isn't the format we want, its pretty specific to each Spark release. Maybe look at some of the other files in this directory and see how they do their usage?

File: examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala [hunk 2]
```diff
@@ -23,6 +33,9 @@ import org.apache.spark.{SparkConf, SparkContext}
 import org.apache.spark.mllib.classification.{NaiveBayes, NaiveBayesModel}
 import org.apache.spark.mllib.linalg.Vectors
 import org.apache.spark.mllib.regression.LabeledPoint
+
+import java.io.File;
```
Comment (holdenk) [inline examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala]: These imports don't seem to be ordered in the way we would want them to be. You should check out https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark and https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide

File: examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala [hunk 3]
```diff
@@ -46,10 +59,18 @@ object NaiveBayesExample {
 
     val predictionAndLabel = test.map(p => (model.predict(p.features), p.label))
     val accuracy = 1.0 * predictionAndLabel.filter(x => x._1 == x._2).count() / test.count()
+    println("model accuracy %f".format(accuracy))
```
Comment (holdenk) [inline examples/src/main/scala/org/apache/spark/examples/mllib/NaiveBayesExample.scala]: We have scala style rules that disable the use of println, you can check out some of the other examples in this directory to see how to disable them when we really want to use println (like here).

**Comment by holdenk** [comment]
Thanks for taking on this issue & fixing it across multiple languages.

A super minor thing is generally we try and have separate JIRAs for separate PRs - so in this case creating a sub task on the initial PR to also update Scala might be better from a tracking point of view (since when I went to look at this PR my first thought was that the issue had already been resolved).

You might also want to try running the scala linter locally on your code (e.g. `./dev/lint-scala`) and make sure it conforms to the style rules.

**Comment by HyukjinKwon** [comment]
Hi @movelikeriver, are you able to proceed this further? If not, maybe it'd be better closed for now.

**Pull request closed without merging**